odxtools/nameditemlist.py

```diff
@@ -69,10 +66,19 @@ def append(self, item: T):
             else:
                 tmp = f"{item_name}_{i}"
 
-    def sort(self, key=None, reverse=False):
+    def sort(self, key: Optional[Callable[[T], str]] = None, reverse: bool = False):
         return self._list.sort(key=key, reverse=reverse)
 
-    def __contains__(self, x) -> bool:
+    def keys(self) -> Iterable[str]:
```

Comment:
What happens if someone have a parameter called "keys" ?

Reply:
it must be accessed via `.keys_` (same as python keywords like `None`, `global`, etc ...)

Reply:
But `keys` is not a keyword